test_conformance/api/test_platform.cpp

```diff
@@ -281,6 +289,15 @@ int test_get_platform_ids(cl_device_id deviceID, cl_context context, cl_command_
       test_error(err, "clReleaseContext failed");
     }
     free(devices);
+
+    err = clGetDeviceIDs(platforms[p], CL_DEVICE_TYPE_DEFAULT, 0, NULL, &num_devices);
+    test_error(err, "clGetDeviceIDs failed.\n");
+    if (num_devices != 1)
+    {
+        log_error("clGetDeviceIDs must return exacly one device\n");
+        total_errors++;
+    }
+    test_assert_error(num_devices == 1, "clGetDeviceIDs must return exacly one device");
```

Comment:
See above - misspelling + possible redundant conditions.  Can we just do this instead?

```suggestion
    if (num_devices != 1)
    {
        log_error("clGetDeviceIDs must return exactly one device\n");
        total_errors++;
    }
```

Reply:
The spelling error still exists in the latest version.